### PR TITLE
data-reset-tool refactored successfully

### PR DESCRIPTION
**Pull request by kialj876**
Signed-off-by: Kial Jinnah <kialj876@gmail.com>

*Issue #:* /bcgov/entity#2261

*Description of changes:*
- refactored to mirror our other services
- linting works
- pytests will work once added
- makefile works the same as legal + colin apis
- should build/deploy in the same way as our other services in openshift
- jenkinsfiles for pipelines to all environments created

Ticket for work that was out of scope: bcgov/entity#2275
- includes fixing linting errors, adding pytests, updating readme, moving it to its own repo

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: data-reset-tool/test/postman/loader.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+						"type": "text"
+					}
+				],
+				"body": {
+					"mode": "formdata",
+					"formdata": [
+						{
+							"key": "file",
+							"type": "file",
+							"src": "/Users/kjinnah/Downloads/CP0001333.xls"
```
Comment (peter-freshworks) [inline data-reset-tool/test/postman/loader.postman_collection.json]: should this be hard-coded? Same for the localhost:5000 above

Comment (kialj876) [reply data-reset-tool/test/postman/loader.postman_collection.json]: I think it's fine for now. No matter what you'll need to change the path to the excel file manually until John creates the scenarios he wants and checks them in. The URL will change once Katie creates the deployment and adds a dev/test/prod url, but I think that should be part of her ticket

Comment (kialj876) [reply data-reset-tool/test/postman/loader.postman_collection.json]: I guess I could check in he excel file I was using and point it to that for now though

**Comment by thorwolpert** [comment]
That's a lot of fixing, cleaning and getting it deployed in OS. Strong work there @kialj876 !!

**Pull request merged**